src/common/modules/LanguageHelper.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Functions useful for evaluating language-related things.
+ *
+ * @public
+ * @module LanguageHelper
+ */
+
+/**
+ * The list of languages the add-on is already translated into.
+ *
+ * If the add-on is only partially translated into one of these langauges and
+ * there may be a lot of strings that are missing it's localized version, it may
+ * be removed from that list here.
+ *
+ * Include the base language and the region-specific version you want to target,
+ * even if you only have a general translation for the base locale.
+ *
+ * @private
+ * @type {string[]}
+ */
+const ADDON_TRANSLATED_INTO = [
+    // German
+    "de", "de-DE",
+].map((lang) => lang.toLowerCase());
```

Comment:
The add-on is of course also in English. I am not sure if that should be included here.

Reply:
Uh yeah. And yes, I’ll include some German translations.